src/app/about/page.js

```diff
@@ -27,7 +27,7 @@ const About = () => {
       {sections.map((section, index) => (
         <div
           key={index}
-          className="flex flex-col mx-auto w-1/2 text-center pb-12"
+          className="flex flex-col mx-auto md:leading-loose w-1/2 md:w-4/5 md:text-xs text-center pb-12"
         >
           <div className="pb-6">{section.title}</div>
           <div className="text-2xl">{section.description}</div>
```

Comment:
text size didn't seem to change in mobile because the CSS is applied to this tag instead of the parent tag, so text-2xl overrides the md:text-xs. you can apply the mobile responsiveness for text size here instead.